### ISSUE TITLE
Gap filling m/z tolerance

### ISSUE THREAD

**Issue by tomas-pluskal** [msdk/msdk#146]
The calculation of m/z tolerance in the gap filler is fishy, needs fixing.
https://github.com/msdk/msdk/blob/master/msdk-features/msdk-features-gapfilling/src/main/java/io/github/msdk/features/gapfilling/GapFillingMethod.java#L172

**Comment by TDI-tenderholt** [comment]
Can you describe how the gap filler is supposed to work? It looks like a list of gaps (unconnected chromatograms?) are found in the feature table, and then filled using targeted detection with a more relaxed m/z tolerance, but it's a bit hard to follow without actually seeing some data.

**Comment by tomas-pluskal** [comment]
Yes, the gap filler finds empty cells in the feature table and then submits their expected m/z & RT to the targeted detection module, which find the missing peaks and fills the gaps. I think it should actually work fine with the commented-out code.

**Comment by tomas-pluskal** [comment]
Fixed
